vnext/PropertySheets/Bundle.props

```diff
@@ -49,15 +48,15 @@
     <BundleEntryFile Condition="'$(BundleEntryFile)' == '' and Exists('$(BundleCommandWorkingDir)\index.windows.js')">index.windows.js</BundleEntryFile>
     <BundleEntryFile Condition="'$(BundleEntryFile)' == ''">index.js</BundleEntryFile>
 
-    <!-- Optional config file for the bundle command (metro will use metro.config.js by default) -->
-    <BundleConfigFile Condition="'$(BundleConfigFile)' == ''"></BundleConfigFile>
-
+    <FullOutDir Condition="'$([System.IO.Path]::GetFullPath($(OutDir)))' == '$(OutDir)'">$(OutDir)</FullOutDir>
```

Comment:
I'm not getting what the desired logic is. Are you checking if the path is relative or not?

Reply:
yes. For C# these seem to be in the form bin\x86\Release, whereas for vcxproj they seem to be full paths. However there is no requirement that this be the case and the doc for this variable says "if it is a relative path, it's relative to the project dir"